math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -11,6 +12,12 @@
 #ifndef ROOT_RVEC
 #define ROOT_RVEC
 
+#if __cplusplus > 201402L
+#define R__RVEC_NODISCARD [[nodiscard]]
+#else
+#define R__RVEC_NODISCARD
+#endif
+
```

Comment:
Why not just use the attribute? Compilers are required to ignore unknown attributes.

Reply:
as per Axel's comment above